### ISSUE TITLE
CKEditor crashes when unmounting: Cannot read property 'destroy' of null

### ISSUE THREAD

**Issue by seyaobey** [ckeditor/ckeditor4-react#56]
## Are you reporting a feature request or a bug?
Apologies if this is not the right place to report my issues. But I thought to write it here to catch the attention of developers. I am integrating your library (thank you for the hard work) into our react application. And `CKEditor` is crashing the app every time it is unmounting.

## Provide detailed reproduction steps (if any)
`CKEditor` is used inside a component within a very large react-typescript enterprise application. Here is how I use it.

```
import * as CKEditor from 'ckeditor4-react';

export const EditorWrapper = ({ editorContent, onChange, onBlur }) => {
   return(
     <CKEditor
          data={editorContent}
          onChange={onChange}
          onBlur={onBlur}
     />
  )
}
```

### Expected result
Every time `EditorWrapper ` is unmounted ( e.g when user goes away from the page where it is used), application should properly unmount `CKEditor`

### Actual result
Application crashes with error:
```
Uncaught TypeError: Cannot read property 'destroy' of null
   at n.value (ckedior.js?3fc:5)
   at ...
```
## Other details
When `CKEditor` is first mounted, I can the following error in the console (which does not crash the application):

```
[CKEDITOR] Error code: editor-incorrect-element ( element: null)
```
I don't understand what this means, what element is null?

Thank you for your help us this component is very critical to our application

* Browser: Chrome Version 75.0.3770.142 (Official Build) (64-bit)
* OS: Windows 10 Prop
* CKEditor version: ckeditor4-react: ^1.0.0-beta.2
* Installed CKEditor plugins: no plugin installed

**Comment by f1ames** [comment]
Hello @seyaobey,
thanks for reporting! I can confirm it's an issue and we already started to work on the fix. The parent ticket is #44 so this one will be solved as well by the fix.

Related #49, #44.

**Comment by asafda** [comment]
Hey @f1ames, any estimation when this fix will be live?

**Comment by ealush** [comment]
Experiencing this as well.

**Comment by f1ames** [comment]
Hello @asafda,
this depends on [upstream issue (in CKEditor 4)](https://github.com/ckeditor/ckeditor-dev/issues/3115), which we are targeting to next CKEditor 4 release (`4.13`, end of September). Then we will need some time to do some testing in integration itself to see if everything works fine. Since we also have other priorities, it's hard to estimate exactly.

One more thing - the integrations are using latest CKEditor 4 version by default (usually released few days after CKEditor 4 release), so if upstream fix is enough (that's our assumption 🤞, however we haven't tested it thoroughly yet), it should be available on the beginning of October (`1.0.0-beta.3`). However, if any adjustments will be needed on the integration side it will have to probably wait for stable release which should be around end of this year.

**Comment by f1ames** [comment]
Fixed in https://github.com/ckeditor/ckeditor4-react/commit/7b34d2c4f896ced08e66359faca28194ed7e8ef4 via https://github.com/ckeditor/ckeditor4/pull/3128.
